plugins/factory-plugin/src/theia-commands.ts

```diff
@@ -117,6 +144,50 @@ export class TheiaCloneCommand {
 
 }
 
+export class TheiaZipCloneCommand implements TheiaCloneCommand {
+
+    private locationURI: string | undefined;
+    private folder: string;
+    private zipfile: string;
+
+    constructor(project: cheApi.workspace.ProjectConfig | cheApi.workspace.devfile.Project, projectsRoot: string) {
+        if (isDevfileProjectConfig(project)) {
+            const source = project.source;
+
+            this.locationURI = source.location;
+            this.folder = path.join(projectsRoot, project.name);
+            const zipfileRE = /[^\/]+(?:\.zip)$/i;
+            const match = this.locationURI.match(zipfileRE);
+            this.zipfile = match ? match[0] : `${project.name}.zip`;
```

Comment:
We can simplify this code 
```
            const zipfileRE = /[^\/]+(?:\.zip)$/i;
            const match = this.locationURI.match(zipfileRE);
            this.zipfile = match ? match[0] : `${project.name}.zip`;
```
because it is just a temporary file, which is going to delete. Just use ```this.zipfile = `${project.name}.zip`;```

Reply:
you're right, will update the PR